src/components/sections/LatticeView.tsx

```diff
@@ -84,6 +84,13 @@ const LatticeView = ({ compilationResult, repeats = 1 }: LatticeViewProps): JSX.
     const slices_len = slices.length
 
     const nodeRef = React.useRef(null) // Add to <Draggable> element
+    const { innerWidth } = window
+    const largeLattice = () => {
+        if (slices_len * (repeats + 1) * cellDimensionPixels > innerWidth - 80) {
```

Comment:
Why does the repeat logic show up here? repeats should only be a feature of the JSON (like a pre-processing step on the JSON)

Reply:
Moving the repeat logic to other places opens the possibility for new and unexpected bugs

Reply:
Until I can easily generate large lattices its unfortunately necessary to test the UI

Reply:
> Something happened with the vw.
> 
> I'm a little concerned about the spreading of repeats code. That's just a feature to helped us debug large lattices. Now I'm not even sure we need it anymore since we can create arbitrarily large JSONs and upload those

Can you provide a means to reproduce this? I was seeing this early on in testing but thought I had fixed that